demos/http/http_demo_plaintext/http_demo_plaintext.c

```diff
@@ -456,34 +427,37 @@ static int sendHttpRequest( HTTPTransportInterface_t * pTransportInterface,
     }
     else
     {
-        LogError( ( "Sending HTTP %s request to %s%s failed with status = %u.",
-                    pMethod, SERVER_HOST, pPath, httpStatus ) );
+        LogError( ( "Failed to send HTTP %s request to %s%s: Error=%s.",
+                    pMethod, SERVER_HOST, pPath, HTTPClient_strerror( httpStatus ) ) );
     }
 
-    if( httpStatus == HTTP_SUCCESS )
-    {
-        return EXIT_SUCCESS;
-    }
-    else
+    if( httpStatus != HTTP_SUCCESS )
     {
-        return EXIT_FAILURE;
+        returnStatus = EXIT_FAILURE;
     }
+
+    return returnStatus;
 }
 
 /*-----------------------------------------------------------*/
 
 /**
  * @brief Entry point of demo.
+ *
+ * This example resolves a domain, then establishes a plaintext TCP connection to
```

Comment:
(Nit) "plaintext TCP" may not be clear to readers. To make it clearer, you can say something "..establishes a TCP connection ~~to~~ with an HTTP server to demonstrate HTTP request/response communication without using an encrypted channel (i.e. without TLS)."

Reply:
Okay I like that description